tests/steps/field/test_field_merge.py

```diff
@@ -0,0 +1,55 @@
+from frictionless import Resource, transform, steps
+
+# Issues
+
+
+def test_step_field_pack_header_907():
+    source = Resource("data/transform.csv")
+    target = transform(
+        source,
+        steps=[steps.field_merge(name="details", from_names=["name", "population"])],
+    )
+    assert target.schema == {
+        "fields": [{"name": "id", "type": "integer"}, {"name": "details"}]
+    }
+
+
+def test_step_field_pack_preserve_header_907():
```

Comment:
I think it would be better to keep metadata/data checks inside one test (the same for `field_pack`)